node-typescript/push-notification-with-fcm/src/main.ts

```diff
@@ -0,0 +1,53 @@
+import { sendPushNotification, throwIfMissing } from './utils.js';
+import admin from 'firebase-admin';
+type Context = {
+  req: any;
+  res: any;
+  log: (msg: any) => void;
+  error: (msg: any) => void;
+};
+
+export default async ({ req, res, log, error }: Context) => {
+  try {
```

Comment:
Let's please copy existing logic from Node version of this function, for sick of consistency.

Reply:
I tried to do less changes and wrote this code as similar to node. if do futher improvement like node then it will give types errors. Please support and inform me next.